addons/advanced_throwing/functions/fnc_prepare.sqf

```diff
@@ -18,6 +18,12 @@
 params ["_unit"];
 TRACE_1("params",_unit);
 
+// Temporarily enable wind info, to aid in throwing smoke grenades effectively
+if (!EGVAR(weather,WindInfo)) then {
```

Comment:
This will be nil if weather isn't loaded.

```suggestion
if !(missionNamespace getVariable [QEGVAR(weather,WindInfo)), false] then {
```

Also, maybe make this a setting enabled by default instead? I don't much care either way but some people might prefer it.

Reply:
When would `weather` not be loaded? Even with ACE Weather and Wind Deflection disabled, the wind indicator can still be enabled.
A setting for the temporary enable sounds reasonable, will do.